### ISSUE TITLE
Figurative-constants for composite fields

### ISSUE THREAD

**Issue by andreioff** [grammarware/software-evolution#21]
The `MOVE` clause support several figurative constants. On Canvas and the BabyCobol website, it is described how these constants behave for field variables, but what about composite fields? Would something like `MOVE SPACES TO A` (where A is a composite field) perform a `MOVE SPACES` to all leaves of A?

**Comment by grammarware** [comment]
Yes, `MOVE SPACES` just fills in the entire target data structure with spaces/zeroes/nulls.
